scripts/install

```diff
@@ -30,6 +31,13 @@ app=$YNH_APP_INSTANCE_NAME
 # CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
 #=================================================
 
+architecture=$(ynh_detect_arch)
+# Check machine architecture (in particular, we don't support ARM and 32bit machines)
+if [ $architecture == "i386" ]
+then
+	ynh_die --message="Sorry, but this app can only be installed on a 64 bits machine :("
```

Comment:
message should be: Sorry, because of nodejs framework, this app can't be installed on i386 (32 bits) machine.